wasmcloud_host/lib/wasmcloud_host_web/live/page_live.html.leex

```diff
@@ -112,98 +112,108 @@
-                    </svg>&nbsp;
-                  </button>
-                </td>
-              </td>
-            </tr>
-          <% end %>
-        </tbody>
-      </table>
+                  <td><%= count %></td>
+                  <td><span class="badge badge-secondary">Running</span></td>
+                  <td>
+                    <button class="btn btn-primary btn-sm" type="button" onClick="navigator.clipboard.writeText('<%= actor %>')" data-toggle="popover" data-trigger="focus" title="" data-content="Copied!">
+                      <%= String.slice(actor, 0..4) %>...
+                      <svg class="c-icon">
+                        <use xlink:href="/coreui/free.svg#cil-copy"></use>
+                      </svg>&nbsp;
+                    </button>
+                  </td>
+                  <td>
+                    <button class="btn btn-primary btn-sm" type="button" onClick="navigator.clipboard.writeText('<%= host_id %>')" data-toggle="popover" data-trigger="focus" title="" data-content="Copied!">
+                      <%= String.slice(host_id, 0..4) %>...
+                      <svg class="c-icon">
+                        <use xlink:href="/coreui/free.svg#cil-copy"></use>
+                      </svg>&nbsp;
+                    </button>
+                  </td>
+                </tr>
+              <% end %>
+            <% end %>
+          </tbody>
+        </table>
+      </div>
     </div>
   </div>
-</div>
-<!-- /.col-->
-<div class="col-xl-6 col-lg-12 col-md-12">
-  <div class="card">
-    <div class="card-header">
-      <div class="row justify-content-between">
-        <div class="col-lg-2 col-md-3 col-sm-3" style="max-width:150px">
-          <h3>Providers</h3>
-        </div>
-        <div class="mfe-2">
-          <div class="dropdown d-inline-block">
-            <button class="btn btn-secondary dropdown-toggle" id="dropdownMenuButton" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Start Provider</button>
-            <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
-              <button class="dropdown-item" data-toggle="modal" data-target="#start_provider_file">From File</button>
-              <button class="dropdown-item" data-toggle="modal" data-target="#start_provider_oci">From Registry</button>
+  <!-- /.col-->
+  <div class="col-xl-6 col-lg-12 col-md-12">
+    <div class="card">
+      <div class="card-header">
+        <div class="row justify-content-between">
+          <div class="col-lg-2 col-md-3 col-sm-3" style="max-width:150px">
+            <h3>Providers</h3>
+          </div>
+          <div class="mfe-2">
+            <div class="dropdown d-inline-block">
+              <button class="btn btn-secondary dropdown-toggle" id="dropdownMenuButton" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Start Provider</button>
+              <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
```

Comment:
Is this `aria` stuff specific to Phoenix, or is this some front-end thing that I just missed out on while working on backends for the past 5 years?

Reply:
`aria` is a set of html attributes that can help make websites more accessible, and a lot of it came prefilled from the coreUI templates. I'm not 100% sure how it ends up interacting with accessibility tools, though.